content/blog/2023/06/22/2023-06-22-mirrors-jenkins-new-IP.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "New Public IPv4 for Jenkins Mirrors"
+tags:
+- infrastructure
+- mirrors
+- jenkins
+authors:
+- dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+links:
+  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
+---
+
+The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.
+
+It provides a "HTTP redirector" service hosted behind the domains `get.jenkins.io`, `mirrors.jenkins.io` and `mirrors.jenkins-ci.org` which has a new public IP since the 12 June 2023.
+
+Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you have a restricted environment.
```

Comment:
```suggestion
[IMPORTANT]
===============================
Please update your DNS servers and firewall rules to the new `20.119.232.75` IP if you have a restricted environment.
===============================
```
Not sure if this syntax is recognised.

Reply:
- The suggestion will not work at all, in any cases: https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/
- Currently trying on local build to see if the standard Asciidoctor admonition works in the jenkins.io website and how it is rendered: if it works i'll update the PR so we can see on the preview site

Reply:
It works with the official syntax: updated the PR